src/nft/pages/collection/index.tsx

```diff
@@ -121,90 +121,94 @@ const Collection = () => {
+                </BannerWrapper>
+                <CollectionDescriptionSection>
+                  {collectionStats && (
+                    <CollectionStats stats={collectionStats || ({} as GenieCollection)} isMobile={isMobile} />
                   )}
-                </Box>
-
-                {/* @ts-ignore: https://github.com/microsoft/TypeScript/issues/34933 */}
-                <AnimatedBox
-                  position={isMobile && (isFiltersExpanded || isBagExpanded) ? 'fixed' : 'static'}
-                  style={{
-                    transform: gridX.to((x) => `translate(${x as number}px)`),
-                    width: gridWidthOffset.to((x) => `calc(100% - ${x as number}px)`),
-                  }}
-                >
-                  {isActivityToggled
-                    ? contractAddress && (
-                        <Activity
-                          contractAddress={contractAddress}
-                          rarityVerified={collectionStats?.rarityVerified ?? false}
-                          collectionName={collectionStats?.name ?? ''}
-                          chainId={chainId}
-                        />
-                      )
-                    : contractAddress &&
-                      collectionStats && (
-                        <Suspense fallback={<CollectionNftsAndMenuLoading />}>
-                          <CollectionNfts
-                            collectionStats={collectionStats || ({} as GenieCollection)}
+                  <div id="nft-anchor" />
+                  <ActivitySwitcher
+                    showActivity={isActivityToggled}
+                    toggleActivity={() => {
+                      isFiltersExpanded && setFiltersExpanded(false)
+                      toggleActivity()
+                    }}
+                  />
+                </CollectionDescriptionSection>
+                <CollectionDisplaySection>
+                  <Box
+                    position={isMobile ? 'fixed' : 'sticky'}
+                    top={{ sm: '0', md: '72' }}
+                    left="0"
+                    width={isMobile ? 'full' : '0'}
+                    height={isMobile && isFiltersExpanded ? 'full' : undefined}
+                    background={isMobile ? 'backgroundBackdrop' : undefined}
+                    zIndex="modalBackdrop"
+                    overflowY={isMobile ? 'scroll' : undefined}
+                  >
+                    {isFiltersExpanded && (
+                      <>
+                        {isMobile && (
+                          <MobileFilterHeader>
+                            <ThemedText.HeadlineSmall>Filter</ThemedText.HeadlineSmall>
+                            <IconWrapper onClick={() => setFiltersExpanded(false)}>
+                              <BagCloseIcon />
+                            </IconWrapper>
+                          </MobileFilterHeader>
+                        )}
+                        <Filters traitsByGroup={collectionStats?.traits ?? {}} />
```

Comment:
minor perf nit: use a constant like like `const EMPTY_OBJ = {}` here so it's not a new prop on ever render